docs/source/install/vagrant.rst

```diff
@@ -6,6 +6,12 @@ It's already pre-installed, configured and tested virtual machine image and so s
 time-consuming installation and configuration steps. Perfect for a quick platform overview,
 testing, demo or even using StackStorm in isolated from the internet air-gapped systems.
 
+.. note::
+
+    The virtual machine image only supports the open source version of StackStorm. Please contact
+    `StackStorm Support <support@stackstorm.com>`_ and let us know if you are interested in an
+    Extreme Workflow Composer (EWC) image or VMWare-compatible virtual appliance.
```

Comment:
This block was under the `Virtual Appliance / OVA` section where `EWC` artifacts mention makes sense.
By moving it close to `Vagrant` section which is a pure OSS-solution adds additional level of confusion.

Reply:
Ok. Please feel free to update it. Thanks.